lib/flutter_webview_plugin.dart

```diff
@@ -1,33 +1,41 @@
 import 'dart:async';
 
 import 'package:flutter/services.dart';
+import 'package:flutter/material.dart';
 
 const _kChannel = 'flutter_webview_plugin';
+const _kEvent = 'flutter_webview_plugin_event';
 
 /// Singleton Class that communicate with a fullscreen Webview Instance
 /// Have to be instanciate after `runApp` called.
 class FlutterWebviewPlugin {
-  final MethodChannel _channel = const MethodChannel(_kChannel);
+  final MethodChannel _channel;
+
+  final EventChannel _event;
+  Stream<String> _stateChanged;
+
+  Stream<String> get stateChanged {
```

Comment:
onUrlChanged Stream already exist for this

Reply:
If there is a difference could you document it ?

Reply:
It is difficult to implement onUrlchanged, I will try this.

Reply:
so what it the purpose of stateChanged ?

Reply:
can you add comment ?
iOS implemented, 
Android not implemented

Reply:
OK